Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -160,6 +160,23 @@ public void CanValidate()
             _fx.Next.Validate();
         }
 
+        [Fact]
+        public void CanDetectInvalidTimestamp()
+        {
+            var invalidBlock = new Block<BaseAction>(
+                index: _fx.Next.Index,
+                difficulty: _fx.Next.Difficulty,
+                nonce: new Nonce(new byte[] { 0x00 }),
+                rewardBeneficiary: _fx.Next.RewardBeneficiary,
+                previousHash: _fx.Next.PreviousHash,
+                timestamp: DateTime.UtcNow + TimeSpan.FromDays(1),
```

Comment:
It would be better to try to falsify the closest case to the boundary; how about testing against 901 seconds later rather than a day later?  It's also a good idea to check if timestamp 899 seconds later is valid.